packages/serve/package.json

```diff
@@ -9,9 +9,6 @@
     "access": "public"
   },
   "license": "MIT",
-  "devDependencies": {
-    "webpack-dev-server": "3.10.3"
```

Comment:
I think we can let this stay here, let's only move the ones used in >1 package

Reply:
only `serve` and `package-utils` have `devDependencies` field in `package.json`.

Reply:
We should move all to root imo, this way we would manage all from same place.

Reply:
Sure it's dev dep so doesn't matter